build.xml

```diff
@@ -1367,6 +1367,7 @@
              more aggressively rather than waiting. See CASSANDRA-14922 for more details.
         -->
         <jvmarg value="-XX:SoftRefLRUPolicyMSPerMB=0" />
+        <jvmarg value="-XX:ActiveProcessorCount=2" />
```

Comment:
I wonder if we should not make it configurable like the ones below `test.cpus`...

Reply:
let's verify if setting this makes any sense first